packages/UI/src/components/atoms/Link/Link.tsx

```diff
@@ -0,0 +1,9 @@
+import type { FC } from 'react';
+import { StyledLink } from './StyledLink';
+import type { LinkProps } from './index';
+
+const Link: FC<LinkProps> = ({ linkTo, linkText }) => {
```

Comment:
let's spread the rest of the props as follows `const Link: FC<LinkProps> = {linkTo, linkText, ...props})`
and use it on the StyledLink like this: `<StyledLink href={linkTo} {...props}>{linkText}</StyledLink>`